scripts/build-babel/index.js

```diff
@@ -0,0 +1,23 @@
+const babel = require('babel-core');
+const fs = require('fs');
+const { execSync } = require('child_process');
+const path = require('path');
+const { log, glob } = require('../utils');
+
+const LIB = 'lib';
+const SRC = 'src';
+
+glob(`${SRC}/**/*.js`)
```

Comment:
Why do this at all? Easier to use Rollup with Babel plugin.

Reply:
Didn't realize. I'll fix it if the babel only scripts stay. Thanks.